docs/ucx/docs/installation/index.mdx

```diff
@@ -0,0 +1,234 @@
+- Network access to your Databricks Workspace used for the [installation process](#install-ucx).
+- Network access to the Internet for [pypi.org](https://pypi.org) and [github.com](https://github.com) from machine running the installation.
+- Databricks Workspace Administrator privileges for the user, that runs the installation. Running UCX as a Service Principal is not supported.
+- Account level Identity Setup. See instructions for [AWS](https://docs.databricks.com/en/administration-guide/users-groups/best-practices.html), [Azure](https://learn.microsoft.com/en-us/azure/databricks/administration-guide/users-groups/best-practices), and [GCP](https://docs.gcp.databricks.com/administration-guide/users-groups/best-practices.html).
+- Unity Catalog Metastore Created (per region). See instructions for [AWS](https://docs.databricks.com/en/data-governance/unity-catalog/create-metastore.html), [Azure](https://learn.microsoft.com/en-us/azure/databricks/data-governance/unity-catalog/create-metastore), and [GCP](https://docs.gcp.databricks.com/data-governance/unity-catalog/create-metastore.html).
+- If your Databricks Workspace relies on an external Hive Metastore (such as AWS Glue), make sure to read [this guide](/docs/reference/external_hms_glue).
+- A PRO or Serverless SQL Warehouse to render the [report](/docs/reference/assessment) for the [assessment workflow](/docs/reference/workflows#assessment-workflow).
+
+Once you [install UCX](#install-ucx), you can proceed to the [assessment workflow](/docs/reference/workflows#assessment-workflow) to ensure
+the compatibility of your workspace with Unity Catalog.
+
+
+
+## Authenticate Databricks CLI
+
+We only support installations and upgrades through [Databricks CLI](https://docs.databricks.com/en/dev-tools/cli/index.html), as UCX requires an installation script run
+to make sure all the necessary and correct configurations are in place. Install Databricks CLI on macOS:
+
+![macos_install_databricks](/img/macos_1_databrickslabsmac_installdatabricks.gif)
+
+Install Databricks CLI on Windows:
+
+![windows_install_databricks.png](/img/windows_install_databricks.png)
+
+Once you install Databricks CLI, authenticate your current machine to a Databricks Workspace:
+
+```commandline
+databricks auth login --host WORKSPACE_HOST
+```
+
+To enable debug logs, simply add `--debug` flag to any command.
+
+
+
+## Install UCX
+
+Install UCX via Databricks CLI:
+
+```commandline
+databricks labs install ucx
+```
+
+You'll be prompted to select a [configuration profile](https://docs.databricks.com/en/dev-tools/auth.html#databricks-client-unified-authentication) created by `databricks auth login` command.
+
+Once you install, proceed to the [assessment workflow](/docs/reference/workflows#assessment-workflow) to ensure the compatibility of your workspace with UCX.
+
+The `WorkspaceInstaller` class is used to create a new configuration for Unity Catalog migration in a Databricks workspace.
```

Comment:
Should this remain here? Looks like implementation details

Reply:
Will be rewritten later, kept it here for sake of transparent copy.